src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        final int numberOfBalls = 3;
+        Ball[] balls = new Ball[numberOfBalls];
+        for (int i = 0; i < numberOfBalls; i++) {
+            balls[i] = new Lottery().getRandomBall();
```

Comment:
we don't need to save balls to array, we can print them right after getting from the method